seisbench/models/base.py

```diff
@@ -1374,6 +1522,20 @@ def detections_from_annotations(annotations, threshold):
 
         return detections
 
+    def get_model_args(self):
```

Comment:
I'd add a call to the super-function to also get the citation.

Reply:
As above, well-spotted and now added in.